features/support/fakebin/man

```diff
@@ -1,2 +1,15 @@
-#!/bin/sh
+#!/bin/bash
 echo man "$@" >> "$HOME"/.history
+
+# Requires man-pages to be built
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+MAN_DIR="$DIR/../../../share/man/man1/"
```

Comment:
I think that the additions to this fake `man` executable are superfluous. The only point of this executable is to log its invocation to `.history` so that the tests can verify that `man` was called and with what arguments. I don't think that it should try have any output or perform any logic.